### ISSUE TITLE
Navigation's jstree markup needs improved semantics for accessibility

### ISSUE THREAD

**Issue by kimisgold** [omeka/omeka-s#2499]
Discussion coming out of https://github.com/omeka/omeka-s/issues/2408. Right now, jstree generates each node as an `<a>` element and contains `<i>` for each button. The anchor does not lead to any locations within the page nor another url, and it shouldn't contain so many interactive elements. The buttons should be actual `<button>`s. The default markup seems baked into the jstree plugin to some extent. We should investigate any improvements in more recent versions of the plugin or consider making our own fork.
